lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -39,6 +39,71 @@
     }
 }
 
+.heading-wrapper-institutions,
+.heading-wrapper-institutions-mobile {
+    background-color: $osf-dark-blue-navbar;
+    color: $color-text-white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 0.5rem;
+
+    label {
+        max-width: 835px;
+        min-width: 250px;
+        width: 50vw;
+        padding-bottom: 0;
+
+        img {
+            display: block;
+            margin: 0 auto;
```

Comment:
They center the desktop and mobile versions of the institution's image. I would ask they stay since they are specific to the PR and will greatly affect layout otherwise.

Reply:
Sounds good